src/main/java/Cuboyd.java

```diff
@@ -0,0 +1,58 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class Cuboyd {
+    // Helpers /////////////////////////////////////////////////////////////////////////////////////////////////////////
+    public static void displayIntroText(){
+        String name = "Cuboyd";
```

Comment:
```suggestion
        String BOT_NAME = "Cuboyd";
```
Should the name of the bot be treated as a constant instead (i.e. be in uppercase)?